### ISSUE TITLE
Document Prometheus Metrics and how to scrape them

### ISSUE THREAD

**Issue by eliobischof** [zitadel/zitadel#6062]
### Preflight Checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe the docs your are missing or that are wrong

As a ZITADEL operator, I'd like to have an overview of ZITADELs metrics and a guide on how to scrape them using Prometheus, so that I can efficiently set up monitoring and alerts.

**Acceptance Criteria**
- [ ] Scrapable metrics are listed in the docs
- [ ] An example scrape configuration is available in the docs
- [ ] An example Prometheus Alertmanager config is available in the docs

**Comment by mffap** [comment]
@amitmavgupta and @matiasracedo this is a related issue to the blog post. Maybe we can, when we anyways publish a blog, also address the docs gaps / or provide a guide for users that look in the docs?

**Comment by amitmavgupta** [comment]
> [@amitmavgupta](https://github.com/amitmavgupta) and [@matiasracedo](https://github.com/matiasracedo) this is a related issue to the blog post. Maybe we can, when we anyways publish a blog, also address the docs gaps / or provide a guide for users that look in the docs?

Agree. Will do and update this PR once the blog is out and also address the docs gaps for Prometheus in the docs site.
